### PR TITLE
Alert wrapper

### PR DESCRIPTION
**Pull request by Atticus29**

### CODE REVIEW

File: src/components/AlertWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  description = '',
+  titleTxt,
+  values,
+  children,
+  ...rest
+}) {
+  if (!titleTxt) {
+    return (
+      <Alert {...rest}>
+        <FormattedMessage id={description} values={values} />
```
Comment (brmscheiner) [inline src/components/AlertWrapper.jsx:18]: `descriptionId`

File: src/components/AlertWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        {children}
+      </Alert>
+    );
+  }
+  return (
+    <Alert {...rest}>
+      <AlertTitle>
+        <Text id={titleTxt} />
```
Comment (brmscheiner) [inline src/components/AlertWrapper.jsx:26]: `title` or `titleId`

Comment (brmscheiner) [reply src/components/AlertWrapper.jsx:26]: `<Text id={titleId}>{title}</Text>`

File: src/components/AlertWrapper.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { FormattedMessage } from 'react-intl';
+import Alert from '@material-ui/lab/Alert';
+// import ResponsiveText from './ResponsiveText';
+import AlertTitle from '@material-ui/lab/AlertTitle';
+import Text from './Text';
+
+export default function AlertWrapper({
+  description = '',
+  titleTxt,
+  values,
```
Comment (brmscheiner) [inline src/components/AlertWrapper.jsx:11]: descriptionValues and titleValues

**Review by brmscheiner — APPROVED** [review]

File: src/components/Alert.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+}) {
+  const isTitle = titleId || titleTxt;
+  return (
+    <Core {...rest} alertChildren={alertChildren}>
+      {isTitle ? (
+        <AlertTitle>
+          {titleId ? (
+            <Text id={titleId} values={titleValues} />
+          ) : (
+            <Text values={titleValues}>{titleTxt}</Text>
```
Comment (brmscheiner) [inline src/components/Alert.jsx:46]: No reason to supply values when the text isn't going to be translated.

File: src/components/Alert.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  children: alertChildren,
+  ...rest
+}) {
+  const isTitle = titleId || titleTxt;
+  return (
+    <Core {...rest} alertChildren={alertChildren}>
+      {isTitle ? (
+        <AlertTitle>
+          {titleId ? (
+            <Text id={titleId} values={titleValues} />
```
Comment (brmscheiner) [inline src/components/Alert.jsx:44]: For some reason I'm not seeing translated titleIds in the UI. For example:
image

File: src/components/profilePhotos/UploadDialog.jsx [hunk 6]
```diff
@@ -4,8 +4,7 @@ import { FormattedMessage } from 'react-intl';
 import 'react-image-crop/dist/ReactCrop.css';
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogContent from '@material-ui/core/DialogContent';
-import Alert from '@material-ui/lab/Alert';
-import AlertTitle from '@material-ui/lab/AlertTitle';
+import CustomAlert from '../Alert';
```
Comment (brmscheiner) [inline src/components/profilePhotos/UploadDialog.jsx:7]: not important, but why call it CustomAlert everywhere?

Comment (Atticus29) [reply src/components/profilePhotos/UploadDialog.jsx:7]: Future-proofing? I figured that someone coming into the code base might take a moment before realizing that our <Alert> was a custom wrapper. So why not just call it that everywhere?

File: src/components/Alert.jsx (hunk 5 above)
Comment (Atticus29) [reply src/components/Alert.jsx:44]: I don't know how I didn't notice this. Maybe my brain was like, "oh, there's a title that's populated. This works!". Good catch.

File: src/components/Alert.jsx (hunk 4 above)
Comment (Atticus29) [reply src/components/Alert.jsx:46]: Did not know that values only mattered for translation. Good to know. Thanks.

File: src/components/profilePhotos/UploadDialog.jsx (hunk 6 above)
Comment (brmscheiner) [reply src/components/profilePhotos/UploadDialog.jsx:7]: Sounds good to me!

**Pull request merged**